docs/CRSF.md

```diff
@@ -26,41 +29,43 @@ The mLRS default parameter settings follow the below recommendations. That is, w
+> [!IMPORTANT]
+> mLRS only supports 400K baud rate.
 
 ## mLRS Tx Module Setup
 
 Set the following parameters using the CLI or Lua script (or OLED if available):
 
 - Tx Ch Source = crsf
-- Tx Ser Baudrate:
-    - 115200 for 19 Hz, 31 Hz, 50 Hz, FSK
-    - 230400 for FLRC (115200 also works)
+- Tx Ser Baudrate: 115200
 - Tx Ser Dest = serial or serial2 (not mbridge!)
 - Tx Snd RadioStat = on
 
-***Note***: Tx Ser Baudrate should be larger than the link data rate in order to provide enough capacity (e.g., in the 50 Hz mode the link data rate is 4100 Bytes/sec and the baudrate should thus be larger than 41000), but otherwise the choice is not critical and largely determined by the user's need. The default value is 115200, which is appropriate for all LoRa modes (50 Hz, 31 Hz, 19 Hz)(and also FSK).
+> [!TIP]
+> The mLRS default settings are Tx Ch Source = crsf, Tx Ser Baudrate = 115200, Tx Ser Dest = serial, Tx Snd RadioStat = on. Therefore, except of Tx Ser Dest, adjustment of the parameters is usually not needed.
 
-The mLRS default settings are Tx Ch Source = crsf, Tx Ser Baudrate = 115200, Tx Ser Dest = serial, Tx Snd RadioStat = on.
+***Notes***: 
+- Tx Ser Baudrate should be larger than the link data rate in order to provide enough capacity (e.g., in the 50 Hz mode the link data rate is 4100 Bytes/sec and the baudrate should thus be larger than 41000), but otherwise the choice is not critical and largely determined by the user's need. The default value is 115200, and is a good choice for most cases.
+- While not necessary, for the FLRC mode it can be benefitial to use 230400.
 
 ## ArduPilot Setup
 
-A basic setup is described in this section which should get one started, further ArduPilot information is available here: [ArduPilot Systems](ARDUPILOT.md).
+A basic setup of the ArduPilot parameters is described in this section which should get you started. Further ArduPilot information is available here: [ArduPilot Systems](ARDUPILOT.md).
 
 ### MAVLink Serial Port
 
-Generally, the baud rate setting is not critical, as mLRS will gracefully work with them all. However, parameter download depends somewhat on your ArduPilot firmware version.
-
-- SERIALx_BAUD:
-    - 115 (= 115200) for 50 Hz, FSK
-    - 57 (= 57600) for 31 Hz
-    - 38 (= 38400) for 19 Hz (57 works very well too, only parameter download is slower with ArduPilot 4.5)
-    - 230 (= 230400) for FLRC
+- SERIALx_BAUD: 57 (= 57600)
 - SERIALx_PROTOCOL = 2 (important, do not use MAVLink v1!)
 - SERIALx_OPTIONS = 4096 (ignore commands from GCS to change stream rates)
 
 ***Note***: 'x' refers to the serial port of your flight controller used for connecting with the mLRS receiver's serial port.
 
+The baud rate setting in SERIALx_BAUD is not critical, as mLRS will gracefully work with them all. However, depending on the ArduPilot firmware version, parameter download can be optimized somewhat by "tuning" the baudrate. For instance, one can try:
```

Comment:
Perhaps, 'However, for ArduPilot version 4.5 or lower, parameter download...'

Reply:
done